Backpack/BottomSheet/Classes/BottomSheet.swift

```diff
@@ -0,0 +1,142 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import FloatingPanel
+
+@objcMembers
+@objc(BPKBottomSheet)
+public final class BottomSheet: NSObject {
+    
+    private struct Constants {
+        static let bottomSheetHeightInHalfPosition: CGFloat = 386.0
+        static let backdropAlpha: CGFloat = 0.3
+    }
+    
+    private lazy var floatingPanelController: BackpackFloatingPanelController = {
+        let panel = BackpackFloatingPanelController(delegate: self)
+        panel.surfaceView.cornerRadius = 24.0
```

Comment:
Well, with our subclass I really didn't wanted to change any default functionality but rather to add something that we need and they don't provide: the ability to add a fixed bottom section that won't scroll.

So, setup is at the instantiation time rather than built-in in our subclass. Do you think I should move it?

Reply:
That sounds reasonable, let's leave it here